src/eval/operation.rs

```diff
@@ -1916,7 +1949,7 @@ impl<R: ImportResolver> VirtualMachine<R> {
                                         id,
                                         String::from("(-$)"),
                                         RichTerm::new(
-                                            Term::Record(RecordData::new(fields, record.attrs)),
+                                            Term::Record(RecordData::new(fields, record.attrs, record.sealed_tail)),
```

Comment:
Can't we do something like `RecordData { fields, ..record }` here? I don't remember if that works for something else than `Default`

Reply:
We can! I have a small personal preference for using `new` because it generally allows us to be more explicit about which parts of a struct should be publicly visible (& would, e.g., reduce the diff if we ever added private fields), but given that isn't applied consistently across the codebase I'm happy to use the "splat" notation here.